src/tool_cb_hdr.c

```diff
@@ -156,7 +156,7 @@ size_t tool_header_cb(char *ptr, size_t size, size_t nmemb, void *userdata)
 
   curl_easy_getinfo(per->curl, CURLINFO_PROTOCOL, &protocol);
   if(hdrcbdata->honor_cd_filename &&
-     (cb > 20) && checkprefix("Content-disposition:", str) &&
+     (cb > 20) && checkprefix("Content-Disposition:", str) &&
```

Comment:
Seems unnecessary.

Reply:
This makes grep by code easier

Reply:
We have `grep -i` for this

Reply:
I consider it superfluous white space edits. The header is case insensitive anyway.